content/doc/book/installing/windows.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: section
+title: Windows
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+= Windows
+
+The simplest way to install Jenkins on Windows is to use the Jenkins Windows installer.
+That program will install Jenkins as a service using a 64 bit JVM chosen by the user.
+Keep in mind that to run Jenkins as a service, the account that runs Jenkins must have permission to login as a service.
+
+include::doc/book/installing/_installation_requirements.adoc[]
+
+== Windows MSI installers
+
+To dowload LTS and latest releases of Windows installer refer to the Windows section of
```

Comment:
`LTS and latest releases of Windows` should be `LTS and the latest releases of Windows`

Reply:
Fixed in 10ac22df09625edd7fb10c3dc47dfddfde09f435

Is the revised phrasing workable @markyjackson-taulia ?